src/engine/index.js

```diff
@@ -257,6 +257,7 @@ export const run = (injectedStore, stats) => {
         sketch = sketches[sketchId]
         const params = getSketchParams(state, sketchId)
         allParams = getSketchParams(state, null, sceneId)
+        allParams.audioTexture = store.getState().inputs.texture.value;
```

Comment:
@funwithtriangles Don't know where/if we landed on this, do we want to just toss the texture in `allParams` for now? Could potentially create an `assets` object too with a bit of a mind towards an asset library that we could just let sketches have full access to . Love the idea of sketches being able to just create weird datamash stuff by swiping from all the textures, models, etc.

Reply:
Thinking it might be better just to expose this as a global? As mentioned above? Seeing as this isn't data that relates to a sketch and is more of a global function? Unless we plan on having this modified per-sketch.

```
const data = window.HEDRON.data

class MySketch {
   ...
   update () {
      this.thingUses(data.audioTexture)
      this.otherThingUses(data.audioData)
   }
}
```